### PR TITLE
diable column both in select list and aggregate functions when using GROUPING SETS/CUBE/ROLLUP

### PR DESCRIPTION
**Pull request by yangzhg**
diable column both in select list and aggregate functions when using GROUPING SETS/CUBE/ROLLUP #2927

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/analysis/SelectStmt.java [hunk 1]
```diff
@@ -349,6 +350,19 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
             }
         }
         if (groupByClause != null && groupByClause.isGroupByExtension()) {
+            for (SelectListItem item : selectList.getItems()) {
```
Comment (kangkaisen) [inline fe/src/main/java/org/apache/doris/analysis/SelectStmt.java]: Please Add a UT. Currently we have a FE UT frame, which could handle SQL analyze bug. you could refer to `DemoTest`.

**Review by kangkaisen — CHANGES_REQUESTED** [review]

File: fe/src/test/java/org/apache/doris/utframe/AnotherDemoTest.java [hunk 2]
```diff
@@ -111,19 +114,25 @@ public static void beforeClass() throws EnvVarNotSetException, IOException,
...
+    @AfterClass
+    public static void TearDown() {
+        try {
+            FileUtils.deleteDirectory(new File(runningDirBase));
+        } catch (IOException e) {
+        }
+    }
+
+    // generate all port from valid ports
+    private static void getPorts() {
```
Comment (kangkaisen) [inline fe/src/test/java/org/apache/doris/utframe/AnotherDemoTest.java]: We could move this method to `UtFrameUtils` ?

Comment (yangzhg) [reply fe/src/test/java/org/apache/doris/utframe/AnotherDemoTest.java]: those ports are class member of DemoTest, and getValidport method already in UtFrameUtils

**Review by kangkaisen — APPROVED** [review]
+1. LGTM

**Pull request merged**